firebase-functions/functions/src/models/game.ts

```diff
@@ -0,0 +1,6 @@
+import { Round } from "./round";
```

Comment:
Wrong double quotes. I'll not go over more, please enable lint and find them :)

Reply:
Thanks, for not going through them :D, all fixed!